analyses/cms-open-data-ttbar/ml.py

```diff
@@ -0,0 +1,302 @@
+import ROOT
```

Comment:
please format this whole file with black (https://black.readthedocs.io/en/stable/index.html) :grimacing:

Reply:
done

Reply:
btw, very nice tool, thank you!